README.md

```diff
@@ -154,6 +154,15 @@ env:
     - RELEASE=true
 ```
 
+You can change the lerna arguments though the LERNA_ARGS variable.
+
+ ```yml
+env:
+  global:
+    - LERNA_ARGS="--force-publish --yes --preid next"
```

Comment:
@douglasduteil should `--yes` is mandatory

Reply:
yes. It's to disable the interactive mode 
see https://github.com/lerna/lerna/tree/master/commands/version#--yes

Reply:
ohhh yes !